### PR TITLE
CMCL-993:  - Doc update for CM3 (screenshots, grammar, spelling, error log check, etc.)

### PR DESCRIPTION
**Pull request by JWalker0007**
### Purpose of this PR

- [x] [CMCL-993](https://jira.unity3d.com/browse/CMCL-993)
**-** Language and grammar pass for new CM3 documentation
**-** Update images/screenshots for CM3
**-** Check error log
**-** Removed the following extensions that were not present in CM3:

1. pixel perfect extension
2. volume settings extension

### Testing status

- [x] Doc successfully built with Package Manager Doc Tools

### CODE REVIEW

File: com.unity.cinemachine/Documentation~/CinemachineBindingModes.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ The offset is interpreted in world space relative to the origin of the Follow ta
 
 Simple follow with world up interprets the offset and damping values in camera-local space. This mode emulates the action a human camera operator would take when instructed to follow a target.
 
-The camera attempts to move as little as possible to maintain the same distance from the target; the direction of the camera with regard to the target does not matter. Regardless of the orientation of the target, the camera tries to preserve the same distance and height from it.
+The camera attempts to move as little as possible to maintain the same distance from the target; the direction of the camera with regards to the target does not matter. Regardless of the orientation of the target, the camera tries to preserve the same distance and height from it.
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/CinemachineBindingModes.md:86]: Hmmm, this one is debatable.  Maybe use a different expression altogether?
https://prowritingaid.com/with-regards-to-vs-with-regard-to#:~:text=With%20Regards%20To-,With%20Regard%20To%20or%20With%20Regards%20To%3F,or%20%E2%80%9Cwith%20relation%20to.%E2%80%9D

File: com.unity.cinemachine/Documentation~/CinemachineFollow.md [hunk 2]
```diff
@@ -14,11 +14,11 @@ The fixed offset can be interpreted in various ways, depending on the Binding Mo
...
-| __Follow Offset__ || The desired offset from the target at which the CmCamera will be positioned. Set X, Y, and Z to 0 to place the camera at the centre of the target. The default is 0, 0, -10, respectively, which places the camera behind the target. |
+| | _Simple Follow With World Up_ | Simple follow with world up interprets the offset and damping values in camera-local space. This mode emulates the action a human camera operator would take when instructed to follow a target. The camera attempts to move as little as possible to maintain the same distance from the target; the direction of the camera with regards to the target does not matter. Regardless of the orientation of the target, the camera tries to preserve the same distance and height from it. |
+| __Follow Offset__ || The desired offset from the target at which the CmCamera will be positioned. Set X, Y, and Z to 0 to place the camera at the center of the target. The default is 0, 0, and -10, respectively, which places the camera behind the target. |
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/CinemachineFollow.md:18]: regard vs regards?  same question

File: com.unity.cinemachine/Documentation~/CinemachineOrbitalFollow.md [hunk 3]
```diff
@@ -22,17 +22,17 @@ If you attach an input controller to the CmCamera, then the player can control t
...
-| | _Simple Tracking With World Up_ | Simple Tracking with world up interprets the offset and damping values in camera-local space. This mode emulates the action a human camera operator would take when instructed to Tracking a target. The camera attempts to move as little as possible to maintain the same distance from the target; the direction of the camera with regard to the target does not matter. Regardless of the orientation of the target, the camera tries to preserve the same distance and height from it. |
+| | _Simple Tracking With World Up_ | Simple Tracking with world up interprets the offset and damping values in camera-local space. This mode emulates the action a human camera operator would take when instructed to track a target. The camera attempts to move as little as possible to maintain the same distance from the target; the direction of the camera with regards to the target does not matter. Regardless of the orientation of the target, the camera tries to preserve the same distance and height from it. |
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/CinemachineOrbitalFollow.md:25]: regard vs regards again.  To me, "regards" feels like "anyways".

File: com.unity.cinemachine/Documentation~/CinemachinePanTilt.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
 # Pan Tilt
 
-This CmCamera __Rotation Control__ behaviour pans and tilts the camera in response stimulus, for instance the user’s input.  This component does not read user input itself; it can be be driven by an [Cinemachine Input Axis Controller](CinemachineInputAxisController.md) component or by some other means that you devise.
+This CmCamera __Rotation Control__ behaviour pans and tilts the camera in response stimulus, for instance the user’s input. This component does not read user input itself; it can be be driven by an [Cinemachine Input Axis Controller](CinemachineInputAxisController.md) component or by some other means that you devise.
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/CinemachinePanTilt.md:3]: behaviour!

File: com.unity.cinemachine/Documentation~/CinemachineUsing.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ The Cinemachine Brain is a component in the Unity Camera itself. The Cinemachine
 
 ## Positioning and Aiming
 
-Use the [__Position Control__ properties](CinemachineVirtualCameraBody.md) in a CmCamera to specify how to move it in the Scene. Use the [__Rotation Control__ properties](CinemachineVirtualCameraAim.md) to specify how to aim it.
+Use the [__Position Control__ properties](CmCamera.md) in a CmCamera to specify how to move it in the Scene. Use the [__Rotation Control__ properties](CinemachineVirtualCameraAim.md) to specify how to aim it.
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/CinemachineUsing.md:38]: Not sure this is the right hotlink.  It should take you somewhere analogous to the second one in the line

File: com.unity.cinemachine/Documentation~/CinemachineVirtualCameraAim.md [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
+Use the Rotation Control behaviors to specify how to rotate the CmCamera. To change the camera’s position, use a [Position Control](CinemachineVirtualCameraBody.md) behaviour.
 
 ![Aim, with the Composer behaviour (red)](images/CinemachineAim.png)
 
 
-Cinemachine includes these behaviours for rotating a CmCamera:
+Cinemachine includes these behaviors for rotating a CmCamera:
 
-- [__None__](CinemachineAimDoNothing.md): Do not procedurally rotate the CmCamera.  The rotation can be controlled by object parenting, or by custom script, or by animation.
+- [__None__](CinemachineAimDoNothing.md): Do not procedurally rotate the CmCamera. The rotation can be controlled by object parenting, a custom script, or by animation.
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/CinemachineVirtualCameraAim.md:10]: This feels inconsistent.  I think you should leave the middle "by" or take out the last one.

File: com.unity.cinemachine/Documentation~/CmCamera.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# CmCamera
+# CmCameras
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/CmCamera.md:1]: This should not be plural.  It's a class name, and we never pluralize them this way

File: com.unity.cinemachine/Documentation~/CmCamera.md [hunk 8]
```diff
@@ -35,17 +35,17 @@ However, the real magic comes when you add Procedural Components to bring the ca
 | | _Orthographic Size_ | When using an orthographic camera, defines the half-height of the camera view, in world coordinates. Available when the Unity camera with the Cinemachine Brain component uses a Projection of Orthographic. |
 | | _Near Clip Plane_ | The closest point relative to the camera where drawing occurs. You can also use [Scene Handles](handles.md) to modify this property.|
 | | _Far Clip Plane_ | The furthest point relative to the camera where drawing occurs. You can also use [Scene Handles](handles.md) to modify this property.|
-| | _Dutch_ | Dutch angle. Tilts the Unity camera on the z-axis, in degrees. This property is unique to the CmCamera; there is no counterpart property in the Unity camera. |
+| | _Dutch_ | A Dutch angle. Tilts the Unity camera on the z-axis, in degrees. This property is unique to the CmCamera; there is no counterpart property in the Unity camera. |
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/CmCamera.md:38]: "The" would be better than "A", I think

File: com.unity.cinemachine/Documentation~/FreeLookCameras.md [hunk 9]
```diff
@@ -1,18 +1,18 @@
-# Free Look Cameras
+# Using FreeLook Cameras
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/FreeLookCameras.md:1]: "Using" doesn't feel right here.  Maybe "Creating"?

File: com.unity.cinemachine/Documentation~/TableOfContents.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 * [About Cinemachine](index.md)
     * [Using Cinemachine](CinemachineUsing.md)
     * [Using CmCameras](CinemachineSetUpVCam.md)
-        * [CmCamera](CmCamera.md)
+        * [CmCameras](CmCamera.md)
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/TableOfContents.md:4]: Should not be plural

File: com.unity.cinemachine/Documentation~/TableOfContents.md [hunk 11]
```diff
@@ -34,9 +34,7 @@
                 * [Cinemachine Confiner2D](CinemachineConfiner2D.md)
             * [Group Framing extension](CinemachineGroupFraming.md)
             * [Follow Zoom extension](CinemachineFollowZoom.md)
-            * [Pixel Perfect extension](CinemachinePixelPerfect.md)
             * [Post Processing extension](CinemachinePostProcessing.md)
-            * [Volume Settings extension](CinemachineVolumeSettings.md)
```
Comment (glabute) [inline com.unity.cinemachine/Documentation~/TableOfContents.md:39]: You need to put these back, they still exist.  PixelPerfect is only available  in URP projects, and VolumeSettings is only available in URP and HDRP projects.

**Review by glabute — CHANGES_REQUESTED** [review]
Comments about the images (can't seem to make them any other way):

CinemachineNoiseProfileb.png: new version showws wrong thing, old version is better
CinemachinePathScene.png: new version needs to show a non-empty path
CinemachineSplash.png: old version is better, even if it's a little outdated.  New version needs to be sexier.
CinemachineStateDrivenCamera.png: Should not show an inspector with a warning.  Give it an animated target.
CinemachineTargetGroup.png: targets chould not be cameras, it's misleading.  Cameras are not legal targets in CM.  Replace them with other GameObjects.

File: com.unity.cinemachine/Documentation~/CinemachinePanTilt.md (hunk 4 above)
Comment (JWalker0007) [reply com.unity.cinemachine/Documentation~/CinemachinePanTilt.md:3]: Thanks! Good catch!

File: com.unity.cinemachine/Documentation~/CinemachineBindingModes.md (hunk 1 above)
Comment (JWalker0007) [reply com.unity.cinemachine/Documentation~/CinemachineBindingModes.md:86]: Hmmm after reading again think you are right and probably should remain as regard. I will rework this one a bit.

**Review by glabute — APPROVED** [review]

**Pull request merged**